ISeriesProvider/MemberAccessor.cs

```diff
@@ -0,0 +1,181 @@
+﻿using LinqToDB.Extensions;
```

Comment:
This is my implementation of a basic epxression based MemberAccessor (fields and props)